pandas/tests/groupby/test_categorical.py

```diff
@@ -48,8 +48,9 @@ def setUp(self):
              'D': np.array(
                  np.random.randn(8), dtype='float32')})
 
-        index = MultiIndex(levels=[['foo', 'bar', 'baz', 'qux'], ['one', 'two',
-                                                                  'three']],
+        index = MultiIndex(levels=[pd.CategoricalIndex(
```

Comment:
you can also simply contruct the MI you need (e.g. from the original issue) directly in the test